### ISSUE TITLE
Promise: Detail page - clarification logs (stretch goal)

### ISSUE THREAD

**Issue by mixth** [wevisdemo/parliament-watch#102]
The PR shall be merged into `promise/main` branch.

# Purpose
To show users what WeVis is doing on clarifying the promise, what are those responses, and how long they are fulfilled.

# Note
- This is a stretch goal issue. We do not need this on our launch date. Please consider this as a low priority item.

# UI
Please refer to [the latest Figma](https://www.figma.com/design/ywiKyd0XIgDXf8crX4aoYE/Design_Phase1-2-%5BPromise-Tracker%5D?node-id=4552-8738&node-type=frame&t=HmpwBOeDR4PxrdVq-0) by the design team.

Clarification Log in action
![In case of no cover image](https://github.com/user-attachments/assets/dcba2212-2598-429b-831a-33228fba7d37)

Multiple statements
![In case of multiple statements](https://github.com/user-attachments/assets/9296890b-3e0a-4925-b014-bb64e40b8841)

Clarified state
![In case of Clarified Promise](https://github.com/user-attachments/assets/23f70580-6d26-4ac1-b0e9-6313d45435d7)

**Comment by svnnynior** [comment]
I will try this out. please feel free to assign it to me.

**Comment by Th1nkK1D** [comment]
> I will try this out. please feel free to assign it to me.

Since it was blocked by another issue, expected to be done within this week, you can create a new component of this section if you want to start working right away.

**Comment by Th1nkK1D** [comment]
@svnnynior good news #113 is merged to `promise/main`, you can rebase to `promise/main` and continue this issue.

**Comment by svnnynior** [comment]
@Th1nkK1D thanks kubbb. just rebased my PR. I have a few questions that i've left it in #112 (nice PR number 😋). could you help take a look?

**Comment by Th1nkK1D** [comment]
> @Th1nkK1D thanks kubbb. just rebased my PR. I have a few questions that i've left it in #112 (nice PR number 😋). could you help take a look?

Sure! I will have a look when I'm free.

(LMAO, the PR number suits you well)
